### ISSUE TITLE
Warning when generating host galaxy postage stamp code

### ISSUE THREAD

**Issue by jbkalmbach** [LSSTDESC/SLSprinkler#16]
Got the following warning when running the code:

```
lensed_hosts/generate_lensed_hosts_sne.py:332: RuntimeWarning: invalid value encountered in double_scalars
 mag_lensed_u = mag_tot_u - 2.5*np.log10(np.sum(g_limage)/np.sum(g_source))
```

```
/global/u1/b/brycek/DC2/sprinkler_mwiesner/lensed_hosts/om10_lensing_equations.py:202: RuntimeWarning: invalid value encountered in less
  img[np.where(R_scale<R_in)] = img_max
/global/u1/b/brycek/DC2/sprinkler_mwiesner/lensed_hosts/om10_lensing_equations.py:205: RuntimeWarning: invalid value encountered in greater
  res[np.where(R_scale>R_out)] = 0.0
```

@mpwiesner Do you know where each of them comes from?

_Originally posted by @jbkalmbach in https://github.com/LSSTDESC/SLSprinkler/pull/12_

**Comment by jbkalmbach** [comment]
I think this is related to the fact that I also found some nans and infs for magnorms in the sne stamp names.

**Comment by jbkalmbach** [comment]
Closed by #14
